developer-docs-site/docs/concepts/txns-states.md

```diff
@@ -59,9 +59,10 @@ A [signed transaction](../guides/sign-a-transaction.md) on the blockchain contai
 - **Expiration time**: A timestamp after which the transaction ceases to be valid (i.e., expires).
 
 ### Types of transactions
-Within a given transaction, the target of execution can be one of two types:
+Within a given transaction, the target of execution can be one of three types:
 
 - An entry point
+- module bundle (publishes multiple modules)
```

Comment:
Module bundle is deprecated, and does not work, which is why it was removed.

Instead, module code is updated via an entry function via a package

Reply:
thanks, I will close this pr.